solidity/contracts/bridge/Bridge.sol

```diff
@@ -1438,15 +1454,13 @@ contract Bridge is Ownable {
     //          `pendingRedemptions` mapping.
     //       4. If request doesn't exist in mapping - revert.
     //       5. If request exits, and is timed out - remove the redemption key
-    //          from `pendingRedemptions` and put it to `invalidRedemptions`.
-    //          No need to check if `invalidRedemptions` mapping already contains
+    //          from `pendingRedemptions` and put it to `timedOutRedemptions`.
+    //          No need to check if `timedOutRedemptions` mapping already contains
     //          that key because `requestRedemption` blocks requests targeting
-    //          non-active wallets so there is no possibility that the given
-    //          redemption key could be reported as timed out multiple times.
-    //          On the other hand, if given redemption key was already
-    //          marked as invalid due to an amount-related fraud, it will not
-    //          be possible to report a time out on it since it won't be
-    //          present in `pendingRedemptions` mapping.
+    //          non-active wallets. Because `notifyRedemptionTimeout` changes
+    //          wallet state after first call (point 9), there is no possibility
+    //          that the given redemption key could be reported as timed out
+    //          multiple times.
```

Comment:
We could add two pieces of information to point 5:

Improve the sentence at the beginning:
> If the request exists, and is timed out - remove the redemption key from `pendingRedemptions` and put it to `timedOutRedemptions` copying the entire `RedemptionRequest` struct there.

And at at the end:
> At the same time, if the given redemption key was already marked as fraudulent due to an amount-related fraud, it will not be possible to report a time out on it since it won't be present in `pendingRedemptions` mapping.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/64395ffafef55a2795e927a1dad3c531d0ef24d8